schema/accepted_schema.sql

```diff
@@ -3582,9 +3582,6 @@ CREATE INDEX IF NOT EXISTS "idx_Users_profileTagIds_deleted_deleteContent_karma"
   "karma"
 );
 
--- Index "idx_Users_walledGardenInvite"
-CREATE INDEX IF NOT EXISTS "idx_Users_walledGardenInvite" ON "Users" USING btree ("walledGardenInvite");
```

Comment:
I think this wants a migration to drop the index in this PR, even if we aren't dropping the actual field on the user object yet

Reply:
Done